docs/reference/ccr/apis/follow/get-follow-info.asciidoc

```diff
@@ -76,7 +76,8 @@ replication options and whether the follower indices are active or paused.
 
 //Begin parameters
 `parameters`::
-(object) An object that encapsulates {ccr} parameters.
+(object) An object that encapsulates {ccr} parameters. This object will be
```

Comment:
Not a big deal, but I usually put the conditional part of the sentence first so folks can stop reading when it doesn't apply to them.
```suggestion
(object) An object that encapsulates {ccr} parameters. If the follower index's status is `paused`,
```